src/quo2/components/list_items/account_list_card/view.cljs

```diff
@@ -0,0 +1,41 @@
+(ns quo2.components.list-items.account-list-card.view
+  (:require
+    [quo2.components.avatars.account-avatar.view :as account-avatar]
+    [quo2.components.icon :as icon]
+    [quo2.components.markdown.text :as text]
+    [quo2.foundations.colors :as colors]
+    [quo2.theme :as quo.theme]
+    [react-native.core :as rn]
+    [quo2.components.list-items.account-list-card.style :as style]))
+
+(defn- internal-view
+  [{:keys [theme account-props networks state action on-press]}]
+  [rn/view
+   {:style               (style/container state theme)
+    :accessibility-label :container}
+   [rn/view
+    {:style {:flex-direction :row
+             :align-items    :center}}
+    [account-avatar/view account-props]
+    [rn/view {:style {:margin-left 8}}
+     [text/text
+      {:weight :semi-bold
+       :size   :paragraph-2} (:name account-props)]
+     [text/text {:size :paragraph-2}
+      (map (fn [network]
+             ^{:key (str network)}
+             [text/text
+              {:size  :paragraph-2
+               :style {:color (get colors/networks network)}} (str (subs (name network) 0 3) ":")])
+           networks)
+      [text/text
+       {:size  :paragraph-2
+        :style {:color (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)}}
+       (:address account-props)]]]]
+   (when (= action :icon)
+     [rn/pressable {:on-press on-press}
+      [icon/icon :i/options
+       {:color               (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)
+        :accessibility-label :icon}]])])
+
+(def view (quo.theme/with-theme internal-view))
```

Comment:
The component doesn't look correct when the OS theme is light, and the `[quo.theme/provider {:theme :dark}]` is used.

That's why I think (ideally) all of our preview screens should use `quo2.theme/provider` and make the `theme` argument part of the preview `descriptor`, it helps designers and devs verify all 4 permutations of *user theme & context theme* are correct.

<img src="https://github.com/status-im/status-mobile/assets/46027/90055d94-d8f9-4886-8498-23a616955212" height="600" />

Reply:
@ilmotta I am not able to reproduce that? Isn't `[quo.theme/provider {:theme :dark}]` is set by the two buttons on top "Set light theme" and "Set dark theme" ?

Reply:
@OmarBasem The theme switcher implemented is not using the theme provider, it's just mutating the global theme ratom.

There are 4 permutations between OS theme and theme provider that we always need to test in components. The one in the screenshot I shared is OS theme light + theme provider dark. Here's the video:

[bug.webm](https://github.com/status-im/status-mobile/assets/46027/2f732627-e91a-4275-8863-2234b1a6368f)

And here's the diff implementing the theme switcher:

<details>

```diff
diff --git a/src/status_im2/contexts/quo_preview/list_items/account_list_card.cljs b/src/status_im2/contexts/quo_preview/list_items/account_list_card.cljs
index ebc09bcb4..16f446dd2 100644
--- a/src/status_im2/contexts/quo_preview/list_items/account_list_card.cljs
+++ b/src/status_im2/contexts/quo_preview/list_items/account_list_card.cljs
@@ -1,12 +1,14 @@
 (ns status-im2.contexts.quo-preview.list-items.account-list-card
   (:require
     [quo2.core :as quo]
+    [quo2.theme :as quo.theme]
     [react-native.core :as rn]
     [reagent.core :as reagent]
     [status-im2.contexts.quo-preview.preview :as preview]))
 
 (def descriptor
-  [{:label   "State:"
+  [preview/theme-option
+   {:label   "State:"
     :key     :state
     :type    :select
     :options [{:key   :default
@@ -34,9 +36,10 @@
                              :action        :none
                              :on-press      (fn [] (js/alert "Button pressed"))})]
     (fn []
-      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
-       [rn/view
-        {:style {:flex               1
-                 :padding-horizontal 20}}
-        [rn/view {:style {:min-height 150}} [preview/customizer state descriptor]]
-        [quo/account-list-card @state]]])))
+      [quo.theme/provider {:theme (:theme @state)}
+       [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+        [rn/view
+         {:style {:flex               1
+                  :padding-horizontal 20}}
+         [rn/view {:style {:min-height 150}} [preview/customizer state descriptor]]
+         [quo/account-list-card @state]]]])))
diff --git a/src/status_im2/contexts/quo_preview/preview.cljs b/src/status_im2/contexts/quo_preview/preview.cljs
index 5bd40231a..58dd6a46d 100644
--- a/src/status_im2/contexts/quo_preview/preview.cljs
+++ b/src/status_im2/contexts/quo_preview/preview.cljs
@@ -237,6 +237,13 @@
                                 {:key k :value (string/capitalize (name k))})))}
           opts)))
 
+(def theme-option
+  {:label   "Override theme:"
+   :key     :theme
+   :type    :select
+   :options [{:key :light :value "Light"}
+             {:key :dark :value "Dark"}]})
+
 (comment
   [{:label "Show error:"
     :key   :error
```
</details>

Reply:
If we don't add the option to override the theme, @Francesca-G won't be able to test all 4 permutations of OS theme + theme provider.